src/Tempest/Http/src/Route.php

```diff
@@ -55,4 +44,24 @@ public function setHandler(MethodReflector $handler): self
 
         return $this;
     }
+
+    /** @return string[] */
+    public static function getRouteParams(string $uriPart): array
+    {
+        $regex = '#\{'. self::ROUTE_PARAM_NAME_REGEX . self::ROUTE_PARAM_CUSTOM_REGEX .'\}#';
+
+        preg_match_all($regex, $uriPart, $matches);
+
+        return $matches[1] ?? [];
+    }
+
+    /** @return string[] */
+    public function routeParts(): array
```

Comment:
This is a utility function used during the Route construction.  it will split the URI on the `/` and thus return all "parts" of the route. These parts are used to build up a routing tree.

Reply:
Maybe rename to `getRouteParts`? Or `split` could also be an option.

Reply:
done, great suggestion